perun-openapi/openapi.yml

```diff
@@ -5487,24 +5817,191 @@ paths:
+    get:
+      tags:
+        - ResourcesManager
+      operationId: getRichResources
+      summary: Get all VO rich resources.
+      parameters:
+        - $ref: '#/components/parameters/voId'
+      responses:
+        '200':
+          $ref: '#/components/responses/ListOfRichResourcesResponse'
+        default:
+          $ref: '#/components/responses/ExceptionResponse'
+
+  /json/resourcesManager/getResourcesCount/vo:
+    get:
+      tags:
+        - ResourcesManager
+      operationId: getResourcesCountForVo
+      summary: Returns number of VO resources
+      parameters:
+        - $ref: '#/components/parameters/voId'
+      responses:
+        '200':
+          $ref: '#/components/responses/IntegerResponse'
+        default:
+          $ref: '#/components/responses/ExceptionResponse'
+
+  /json/resourcesManager/getResourcesCount/all:
+    get:
+      tags:
+        - ResourcesManager
+      operationId: getResourcesCountForAll
+      summary: Gets count of all users.
+      responses:
+        '200':
+          $ref: '#/components/responses/IntegerResponse'
+        default:
+          $ref: '#/components/responses/ExceptionResponse'
+
+  /json/resourcesManager/deleteAllResources:
+    get:
+      tags:
+        - ResourcesManager
+      operationId: deleteAllResources
+      summary: Deletes all VO resources
+      parameters:
+        - $ref: '#/components/parameters/voId'
+      responses:
+        '200':
+          $ref: '#/components/responses/VoidResponse'
+        default:
+          $ref: '#/components/responses/ExceptionResponse'
+
+  /json/resourcesManager/getAllowedResources:
+    get:
+      tags:
+        - ResourcesManager
+      operationId: getAllowedResources
+      summary: Get all resources which have the member access on.
+      parameters:
+        - $ref: '#/components/parameters/memberId'
+      responses:
+        '200':
+          $ref: '#/components/responses/ListOfResourcesResponse'
+        default:
+          $ref: '#/components/responses/ExceptionResponse'
+
+  /json/resourcesManager/createResourceTag/resourceTag:
+    post:
+      tags:
+        - ResourcesManager
+      operationId: createResourceTagWithResourceTag
+      summary: Create new resource tag in VO
+      responses:
+        '200':
+          $ref: '#/components/responses/ResourceTagResponse'
+        default:
+          $ref: '#/components/responses/ExceptionResponse'
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              title: InputCreateResourceTagWithResourceTag
+              description: "input to createResourceTagWithResourceTag"
+              type: object
+              required:
+                - resourceTag
+                - vo
+              properties:
+                resourceTag: { $ref: '#/components/schemas/ResourceTag' }
+                vo: { type: integer, description: 'vo id' }
+
+  /json/resourcesManager/createResourceTag/tagName:
+    get:
```

Comment:
This operation changes the state at the server, so it should use POST HTTP method, not GET. It will need /urlinjsonout/ prefix  to keep the URL parameters.